### PR TITLE
*: fix data races detected by go test -race

### PR DESCRIPTION
**Pull request by coocood**

### CODE REVIEW

File: server/server.go [hunk 1]
```diff
@@ -68,7 +68,11 @@ type Server struct {
 
 // ConnectionCount gets current connection count.
 func (s *Server) ConnectionCount() int {
-	return len(s.clients)
+	var l int
```
Comment (ngaut) [inline server/server.go]: s/l/cnt/

**Comment by shenli** [comment]
@disksing PTAL

**Comment by disksing** [comment]
LGTM

**Comment by shenli** [comment]
LGTM

**Pull request merged**